src/common/components/grid/cell-renderer/product-type.cell-renderer.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import React from 'react';
+import { ICellRendererParams } from 'ag-grid-community';
+import { IconButton, Tooltip } from '@map-colonies/react-core';
+import { Box } from '@map-colonies/react-components';
+import {
+  JobModelType,
+  ProductType,
+} from '../../../../discrete-layer/models';
+
+interface IEntityTypeCellRendererParams extends ICellRendererParams {
+  style?: Record<string, unknown>;
+}
+
+export const ProductTypeRenderer: React.FC<IEntityTypeCellRendererParams> = (
+  props
+) => {
+  const type: ProductType = ((props.data as JobModelType).productType as ProductType);
+
+
+  const {
```

Comment:
We should talk about it, in any case ProductType related info should be on upper level